src/components/screen/index.jsx

```diff
@@ -60,80 +62,110 @@ class Screen extends React.Component {
+            {sidenavHeader !== null
+            && (
+              <div className="screen-sidenav-header">
+                {sidenavHeader && <h3 className="screen-sidenav-header-title">{sidenavHeader}</h3>}
+                <Button outline color="hover" className="screen-sidenav-toggler" onClick={this.toggleSidenav}>
+                  <Icon imgSrc="arrow-left" alt="close" className="screen-sidenav-toggler-img" />
+                </Button>
+              </div>
+            )}
+            <div className="screen-sidenav-body">{sidenav}</div>
+          </div>
+        )}
 
         <div className="screen-main">
-          {(header || primaryActions || breadcrumbs.length) &&
-          <div className="screen-header-container">
-            {breadcrumbs.length > 0 &&
-            <h1 className="screen-header-title">
-              <Breadcrumb>
-                {breadcrumbs.map(breadcrumb => (
-                  <BreadcrumbItem key={breadcrumb.text}>
-                    {breadcrumb.to && <RouterLink to={breadcrumb.to}>{breadcrumb.text}</RouterLink>}
-                    {!breadcrumb.to && breadcrumb.text}
-                  </BreadcrumbItem>
-                ))}
-              </Breadcrumb>
-            </h1>}
-            {breadcrumbs.length === 0 && header && <h1 className="screen-header-title">{header}</h1>}
-            {primaryActions && <ScreenPrimaryActions>{primaryActions}</ScreenPrimaryActions>}
-          </div>}
+          {(header || primaryActions || breadcrumbs.length)
+          && (
+            <div className="screen-header-container">
+              {breadcrumbs.length > 0
+              && (
+                <h1 className="screen-header-title">
+                  <Breadcrumb>
+                    {breadcrumbs.map(breadcrumb => (
+                      <BreadcrumbItem key={breadcrumb.text}>
+                        {breadcrumb.to && <RouterLink to={breadcrumb.to}>{breadcrumb.text}</RouterLink>}
+                        {!breadcrumb.to && breadcrumb.text}
+                      </BreadcrumbItem>
+                    ))}
+                  </Breadcrumb>
+                </h1>
+              )}
+              {breadcrumbs.length === 0 && header && <h1 className="screen-header-title">{header}</h1>}
+              {primaryActions && <ScreenPrimaryActions>{primaryActions}</ScreenPrimaryActions>}
+            </div>
+          )}
 
           <AlertBar alerts={alerts} onDismiss={id => dispatch(dismissAlert(id))} />
 
-          {tabs.length > 0 &&
-          <Nav underline className="screen-navtabs">
-            {tabs.map(tab => (
-              <NavItem key={tab.to}>
-                <RouterLink navTab to={tab.to}>{tab.text}</RouterLink>
-              </NavItem>
-            ))}
-          </Nav>}
+          {tabs.length > 0
+          && (
+            <Nav underline className="screen-navtabs">
+              {tabs.map((tab, i) => {
+                let className = '';
+                if (i === 0 && !get(match, 'params.tab')) {
```

Comment:
@weiwang314 why not just one line `const className = i === 0 && !get(match, 'params.tab') ? 'active' : '';`